### DISCUSSION TITLE
Feature Inquiry for Multiple URL Linking

### DISCUSSION THREAD

**Discussion by frans-delacruz** [beyondcode/herd-community#1206] [Q&A]
Hi Folks,

I would like to inquire if there is a feature that allows linking a site with multiple URLs, such as the following, where I can incorporate different country names in my application:

http://country.test/vietnam
http://country.test/thailand
Could you kindly let me know if this is possible or if there's a specific approach for achieving this?

Thank you in advance!

**Comment by mpociot** [comment]
Hey!

This is nothing that Herd handles on its own.

Your application should accept those individual request parameters.

For example, when using a framework like Laravel, you would need to define a route that looks like this:

```php
Route::get('/{country}', function ($country) {
    echo "You are requesting: ".$country;
});
```

**Reply by frans-delacruz** [reply]
Hi @mpociot ,

Sorry about my explanation im trying to use herd w/ codeigniter 3. to follow up my explanation in apache config i can do redirect this country in same application but since herd has nginx i still dont know how to config it.


```
<VirtualHost *:80> 
    DocumentRoot "application/code"
    ServerName country.test
    ServerAlias *.country.test
    <Directory "application/code">
        AllowOverride All
        Require all granted
    </Directory>
    Alias /vietnam "application/code"
	<Directory "application/code">
		Header set Access-Control-Allow-Origin "*"
		Header set Access-Control-Allow-Headers "*"
		Header set Access-Control-Allow-Methods "*" 
		Options Indexes FollowSymLinks
		AllowOverride All
		Require all granted
	</Directory>

    Alias /thailand "application/code"
	<Directory "application/code">
		Header set Access-Control-Allow-Origin "*"
		Header set Access-Control-Allow-Headers "*"
		Header set Access-Control-Allow-Methods "*" 
		Options Indexes FollowSymLinks
		AllowOverride All
		Require all granted
	</Directory>

</VirtualHost>
```
